bentoml/_internal/frameworks/pytorch_lightning.py

```diff
@@ -11,21 +11,19 @@
 from ...exceptions import NotFound
 from ...exceptions import MissingDependencyException
 from ..models.model import Model
+from .common.pytorch import torch
```

Comment:
ugh I we don't want to import module like this. I think it is ok to have `import torch` like below.